### DISCUSSION TITLE
Table or Data table Component

### DISCUSSION THREAD

**Discussion by twokodes** [svelteuidev/svelteui#339] [Feature Requests]
Hello

Thanks for all the hardwork 

Will a data table with pagination be added to the list of components?

**Comment by BeeMargarida** [comment]
Pagination component is on the roadmap. A table component is not, but can be made in the future.

**Comment by flaime** [comment]
Really nice job so far!
I wonder why a table component is not on the roadmap as it seems to me like a fairly "basic" element?

**Reply by BeeMargarida** [reply]
Added!
